eve/main.yml

```diff
@@ -568,6 +579,10 @@ stages:
         <<: *configure_end2end_tests
         env:
           <<: [*kind-env, *e2e-env, *http-env, *secrets-env]
+    - ShellCommand:
+        <<: *run_iam_policies_test
+        env:
+          <<: [ *kind-env, *e2e-env, *http-env, *secrets-env ]
```

Comment:
Just asking, not requesting any change here: do we want to run those tests in the `end2end-sharded` environment?

Reply:
no, we won't, it will be removed right now in Killian's pr. sorry to let you switch around ;)